tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = packaging, py27, py36, pep8, check_isort
+envlist = packaging, py35, py36, py37, pep8, check_isort
```

Comment:
Do we need py36? Can we just do py35 + py37?

Reply:
This is just for what happens when you run `tox` with no arguments, and fills in what's available when you do `tox -l`, so people can see what envs exist.